tests/unittests/GraphOptzTest.cpp

```diff
@@ -203,11 +203,15 @@ TEST_F(GraphOptz, optimizeBatchNormAfterConv) {
   EXPECT_EQ(F_->getNodes().size(), 3);
 
   ::glow::convertPlaceholdersToConstants(F_, bindings_, {});
-  ::glow::optimize(F_, CompilationMode::Infer);
-  EXPECT_EQ(F_->getNodes().size(), 2);
+  optimizedF_ = optimizeFunction(F_);
+  EXPECT_EQ(optimizedF_->getNodes().size(), 2);
 
-  ASSERT_EQ(A->getNumUsers(), 1);
-  Node *newCV = A->getUsers().begin()->getUser();
+  bindings_.allocate(mod_.getPlaceholders());
+  bindings_.get(A)->getHandle().randomize(-1.0, 1.0, mod_.getPRNG());
+  checkNumericalEquivalence();
```

Comment:
Nit: I think the rest of these tests make post-optimization asserts on the graph first and then perform this numerical equivalence check. Let's move this after the asserts below for the sake of consistency.

Reply:
Done.